situp-plugins/elasticsearch/src/main/resources/otel-v1-apm-span-index-template.json

```diff
@@ -3,12 +3,19 @@
     "date_detection": false,
```

Comment:
events is actually array of objects which contains a field called `time`. Now should we enable this flag or define the mapping events with `time` as `date_time_nanos`

Reply:
Agreed.
```
{
  "time": "2020-11-03T00:01:19.277324Z",
  "name": "",
  "attributes": {
    "method": "GET",
    "level": "info",
    "event": "HTTP request received",
    "url": "/customer?customer=392"
  },
  "droppedAttributesCount": 0
}
```

Reply:
Maybe also enable the flag as there might be other fields fit into datetime

Reply:
Fixed

Reply:
I think we shouldn't enable the flag we are explicitly mapping them. I am sure date_detection is an unwanted cost. So lets disable detection as we will going forward explicitly map date_time.